GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -341,15 +329,13 @@ void TextureCacheVulkan::StartFrame() {
 			// Since textures are 2D maybe we should square this, but might get too non-aggressive.
 			slabPressureLimit *= g_Config.iTexScalingLevel;
 		}
-		Decimate(allocator_->GetSlabCount() > slabPressureLimit);
+		Decimate(false);  // TODO: Use some indication from VMA.
```

Comment:
This is interesting:
https://gpuopen-librariesandsdks.github.io/VulkanMemoryAllocator/html/staying_within_budget.html#staying_within_budget_querying_for_budget

Seems like something good to mess with later, I suppose.

-[Unknown]